### ISSUE TITLE
Line chart does not reflect data with decimals

### ISSUE THREAD

**Issue by statgreen** [statisticssweden/PxWeb#153]
**Describe the bug**
Saved query below extracts a table with decimals (precision=2) and generates a line chart in steps, not using the decimals

this is pxweb2019v1, but can also be reproduced on 2021v1

**To Reproduce**
1. Go to https://bank.stat.gl:443/sq/fe551c5d-56b6-4269-89d8-3acff776a2cf

**Expected behavior**
a non-stepwise line chart

**Actual behavior**
stepwise line chart , not reflecting decimal values

**Screenshots**
use saved query

**Environment information**
 - OS: [win]
 - Browser [chrome]

**Comment by likp** [comment]
Looks like it is rounding to integer values.

**Comment by larpSTATGL** [comment]
Yes it is still rounding to integer. I have tried on demosite version 2022, today, and believe it is still a bug
![image](https://user-images.githubusercontent.com/14994906/223652343-9a21a80e-a3d7-4796-8567-06574dbbbed0.png)
